src/kafkaaggregator/testtopic/agents.py

```diff
@@ -0,0 +1,157 @@
+"""Faust agent for the test topic aggregation example.
+
+The Faust agent persists the messages for the test topic in tumbling windows.
+See https://faust.readthedocs.io/en/latest/userguide/tables.html#windowing
+
+The window_size parameter controls the number of messages to aggregate,
+if the test topic is produced at a constant frequency, the number of messages
+aggregated in each window is n = window_size * frequency. The window range is
+relative to the time field in the test topic.
+
+The window_expires parameter controls when the callback function to
+process the expired window(s) is called. When a window is processed, a new
+message for the aggregated test topic is produced with the aggregated results.
+"""
+
+__all__ = ["aggregate", "process_window", "process_stream"]
+
+
+import logging
+from statistics import mean
+from typing import AsyncGenerator, List, Tuple
+
+from faust import web
+from faust.types import StreamT
+
+from kafkaaggregator.app import app
+from kafkaaggregator.config import Configuration
+from kafkaaggregator.testtopic.models import AggTestTopic, TestTopic
+
+logger = logging.getLogger("kafkaaggregator")
+
+config = Configuration()
```

Comment:
I wonder if the `kafkaaggregator.app` module could have a config instance in it so that individual modules don't need to re-evaluate Configuration from the environment.